BitbucketServerAutomation/Functions/Get-BBServerFileContent.ps1

```diff
@@ -0,0 +1,68 @@
+# Copyright 2016 - 2018 WebMD Health Services
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function Get-BBServerFileContent
+{
+    <#
+    .SYNOPSIS
+    Gets the raw content of a file in a repository.
+    
+    .DESCRIPTION
+    The `Get-BBServerFileContent` function gets the raw content of a file from a repository. Pass the connection to the
+    Bitbucket Server to the `Connection` object. Pass the project key of the repository to the `ProjectKey` parameter.
+    Pass the repository name to the `RepoName` parameter. Pass the path to the file in the repository to the `Path` 
+    parameter. The raw content of the file is returned.
+
+    To retrieve the value of the file at a specific commit, pass the commitish identifier (a tag, hash, branch name, 
+    etc.) to the `Commitish` parameter.

```

Comment:
Without `Commitish` does it default to the latest commit on the repositories configured default branch?

Reply:
🤷